hammerdb/azuredeploy.json

```diff
@@ -248,6 +249,20 @@
                             "priority": 101,
                             "direction": "Inbound"
                         }
+                    },
+                    {
+                        "name": "customSshPort",
```

Comment:
this might deserve a comment like:

add security rule to enable custom ssh port

Reply:
You mean a separate comment other than the current description 2 lines below or update the description below?

Reply:
> update the description below?

makes more sense, maybe:

"description": "allow accessing to custom ssh port from all ip's",